src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -620,7 +617,17 @@ li {
   padding: 0.25rem;
 }
 
-::v-deep .theme--light.v-label--is-disabled {
-  color: white !important;
- }
+.message-box {
+  background-color: $BCgovGold0;
+  border-color: $BCgovGold5;
+  border-radius: 0;
+
+  p {
+    margin: 1.25rem;
+    padding: 0;
+    font-size: 0.875rem;
+    letter-spacing: 0.01rem;
+    color: $gray7;
+  }
+}
```

Comment:
If we make use of this again then we should move this to the base assets file or something.